test/app/subscriptions.go

```diff
@@ -39,7 +39,27 @@ var allSubjects = map[apps.Subject]apps.Expand{
 		Team:       apps.ExpandAll,
 		TeamMember: apps.ExpandAll,
 	},
-	// apps.SubjectBotMentioned: {
+	apps.SubjectSelfJoinedChannel: {
+		User:          apps.ExpandAll,
+		Channel:       apps.ExpandAll,
+		ChannelMember: apps.ExpandAll,
+	},
+	apps.SubjectSelfLeftChannel: {
+		User:          apps.ExpandAll,
+		Channel:       apps.ExpandAll,
+		ChannelMember: apps.ExpandAll,
+	},
+	apps.SubjectSelfJoinedTeam: {
+		User:       apps.ExpandAll,
+		Team:       apps.ExpandAll,
+		TeamMember: apps.ExpandAll,
+	},
+	apps.SubjectSelfLeftTeam: {
+		User:       apps.ExpandAll,
+		Team:       apps.ExpandAll,
+		TeamMember: apps.ExpandAll,
+	},
+	// apps.SubjectSelfMentioned: {
```

Comment:
could we remove the code? we can recover it with `git log -p`

Reply:
@mickmister @DHaussermann I am maintaining my view that functionality that is covered by restapitest should be excluded from the test app to minimize its complexity and maintenance. Why do we need subscriptions in the test app?